repos/fdbt-types/matchingJsonTypes.ts

```diff
@@ -237,6 +238,7 @@ export interface SingleTicket extends BasePointToPointTicket {
     fareZones: FareZone[];
```

Comment:
we need to add to WithBaseIds<T> type, like this

export type WithBaseIds<T> = Omit<
    T,
    | 'passengerType'
    | 'ageRange'
    | 'ageRangeMin'
    | 'ageRangeMax'
    | 'proof'
    | 'proofDocuments'
    | 'timeRestriction'
    | 'fareDayEnd'
> & { passengerType: { id: number }; timeRestriction?: { id: number }; cap?: { id: number } };